MODULE.bazel

```diff
@@ -17,9 +17,20 @@ bazel_dep(name = "rules_bison", version = "0.3")
 
 # Last versions compatible w/ bazel6 https://github.com/chipsalliance/verible/issues/2336
 bazel_dep(name = "abseil-cpp", version = "20240116.2")
-bazel_dep(name = "protobuf", version = "28.3")
+bazel_dep(name = "protobuf", version = "30.2")
 
 bazel_dep(name = "googletest", version = "1.14.0.bcr.1", dev_dependency = True)
 
 # To build compilation DB and run build-cleaning
 bazel_dep(name = "bant", version = "0.2.0", dev_dependency = True)
+
+# Make it possible to use rules_python as root (needed for the CI). See
+# https://github.com/bazel-contrib/rules_python/pull/713 for more details.
+bazel_dep(name = "rules_python", version = "1.3.0")
+
+python = use_extension("@rules_python//python/extensions:python.bzl", "python")
```

Comment:
Where do we need Python, and is there a way to avoid it ?

Reply:
Some rule from Kythe extractors.bzl needed it. But I rolled out the extractors.bzl and took only what's needed to extract Verible. Now it works.